src/api/http.js

```diff
@@ -1,9 +1,12 @@
 import axios from 'axios';
 
+const URL = 'https://social-network.samuraijs.com/api/1.0/';
+const KEY = 'eeef07f4-5659-4447-a94e-52a1e5f5b9ba';
```

Comment:
For future, let's move your personal credentials into .env so no one will see it =)
You can get it using: process.env.KEY

Reply:
Done!